omod/src/main/resources/config.xml

```diff
@@ -18,15 +18,11 @@
 
     <require_modules>
         <require_module version="${emrapiVersion}">org.openmrs.module.emrapi</require_module>
-        <require_module version="${paperrecordVersion}">org.openmrs.module.paperrecord</require_module>
```

Comment:
Where was this paperrecordVersion even coming from?  I don't see a line to remove it from the pom.

Reply:
Yeah, it's a good point, I noticed this and was going to note that this suggests this line was in error (and that variable wasn't being replaced) and OMRS startup was just silently ignoring the error, but I didn't dig deeper.